services/server/server.go

```diff
@@ -236,6 +248,11 @@ type Server struct {
 
 // ServeGRPC provides the containerd grpc APIs on the provided listener
 func (s *Server) ServeGRPC(l net.Listener) error {
+	ctx := context.Background()
+	span := trace.SpanFromContext(ctx)
+	_, grpcSpan := (span.Tracer().Start(ctx, "grpcSpan"))
```

Comment:
This should use `otel.Tracer("tracer_name")` (or, perhaps better, a `Tracer` included in the `Server` struct created from a user-provided `TracerProvider` or the global `TracerProvider` if none is provided) instead since `context.Background()` will never contain an active span you're only going to get a no-op `Tracer` from this.  Some of the [guidance we provide for -contrib instrumentation](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation#contents) is probably also applicable here.